tests/stress/config.toml

```diff
@@ -0,0 +1,6 @@
+[init]
+genesis = "/opt/gocode/src/github.com/ChainSafe/gossamer/node/gssmr/genesis.json"
```

Comment:
this need to go, I could not figure a way to make it work with the cmd line yet

Reply:
if it is empty following the merge of #795, it will use the gssmr genesis file by default

it also looks like you are using the full expanded path, either you need to do this because you are calling gossamer commands from different directories or this is not necessary and you can you a local path (ie, `.../.../node/gssmr/genesis.json`)

Reply:
there are also a couple utility method for this, which can be used now and get updated in #795 as well (ie, `utils.GetGssmrGenesisPath` and `utils.GetKsmccGenesisPath`)

Reply:
oh, good point. I actually dont need this file any longer. So I removed it.